packages/sqs/lib/sqs/AbstractSqsConsumer.ts

```diff
@@ -170,12 +172,21 @@ export abstract class AbstractSqsConsumer<
 
   public async start() {
     await this.init()
+    if (this.consumer) this.consumer.stop()
+
+    const visibilityTimeout = await getQueueAttributes(
+      this.sqsClient,
+      { queueUrl: this.queueUrl },
+      ['VisibilityTimeout'],
+    )
```

Comment:
`VisibilityTimeout` should be picked from the queue to support `queueLocator` case

Reply:
Can we avoid this extra lookup in case we are not using `queueLocator`, so that we do not do extra hop? If we are creating a new queue anyway, we can rely on its visibility timeout.

Reply:
yeah, will add that piece of logic 🙇